src/mantine-core/src/MultiSelect/MultiSelect.tsx

```diff
@@ -324,6 +324,10 @@ export const MultiSelect = forwardRef<HTMLInputElement, MultiSelectProps>((props
       valuesOverflow.current = true;
       setDropdownOpened(false);
     }
+
+    if (filteredData.length === 0) {
```

Comment:
Will this work correct with creatable prop?

Reply:
Yes vitaly, i have tested creatable multiselect and they are working fine

Reply:
Great, thanks!